src/main/scala/scalaz/reactive/Event.scala

```diff
@@ -1,11 +1,37 @@
 package scalaz.reactive
+import scalaz.Monoid
+import scalaz.zio.{ IO, RTS }
 
-case class Event[+A](value: Future[Reactive[A]]) extends AnyVal {
+case class Event[A](value: Future[Reactive[A]]) extends RTS { self =>
+
+  def merge(
+    v: Event[A]
+  ): Event[A] = { // if Event[+A]: covariant type A occurs in contravariant position
```

Comment:
Let's not remove variance, the way it is done is by providing events of a supertype of A, so the type signature of merge must be:
`def merge[AA >: A](v: Event[AA]): Event[AA]`

Reply:
fixed, thanks